tests/e2e-openshift-ossm/ossm-tempostack-otel/install-otel-collector.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: opentelemetry.io/v1alpha1
+kind: OpenTelemetryCollector
+metadata:
+  name: simplest
+  namespace: tracing-system
+spec:
+  config: |
+    receivers:
+      zipkin:
```

Comment:
Could you add some verification that we can receive GRPC and HTTP `otlp` traces from the workload?

Reply:
Done, the test generates traces from a project call otlp-app using telemetrygen and sends it to the OTEL collector in tracing-system to both OTLP HTTP and GRPC. Then the traces for both protocols with the servicename telemetrygen-http and telemetrygen-grpc is verified in Tempo Monolithic instance.

Reply:
https://github.com/grafana/tempo-operator/pull/866/files#diff-a84cef7557afc667fd2929ec6aae5722987b299a6a983534bcdd1f08e02c7ca1R64

Reply:
Done, added for verification of traces sent via OTLP GRPC an HTTP in TempoStack with OTEL test case as well.